src/librustc_errors/diagnostic_builder.rs

```diff
@@ -177,6 +177,11 @@ impl<'a> DiagnosticBuilder<'a> {
                                               found_extra: &dyn fmt::Display,
                                               ) -> &mut Self);
 
+    forward!(pub fn note_unsuccessfull_coercion(&mut self,
```

Comment:
Nit: I think it is spelled `unsuccessful` (with one `l`), at least in American English.